File: MyClassLibrary/Forms/XtraForm1.vb
Namespace Forms

    Public Class XtraForm1

        Private Sub AlertControl_AlertClick(sender As Object, e As DevExpress.XtraBars.Alerter.AlertClickEventArgs) Handles AlertControl.AlertClick
            Classes.class_Procedures.Copy_Clipboard(e.Info.Text)
        End Sub

    End Class

End Namespace

File: MyClassLibrary/Forms/XtraForm1.cs
namespace MyClassLibrary.Forms
{
	public partial class XtraForm1
	{
		public XtraForm1()
		{
			InitializeComponent();
		}

		private void AlertControl_AlertClick(object sender, DevExpress.XtraBars.Alerter.AlertClickEventArgs e)
		{
			Classes.class_Procedures.Copy_Clipboard(e.Info.Text);
		}
	}
}